src/Components/Login.js

```diff
@@ -0,0 +1,142 @@
+                </Form.Group>
+                <Form.Group controlId="exampleForm.ControlInput1">
+                  <Form.Label>PASSWORD</Form.Label>
+                  <InputGroup>
+                    <InputGroup.Prepend>
+                      <InputGroup.Text className="inputIcon">
+                        <svg className="bi bi-lock" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                          <path fill-rule="evenodd" d="M11.5 8h-7a1 1 0 0 0-1 1v5a1 1 0 0 0 1 1h7a1 1 0 0 0 1-1V9a1 1 0 0 0-1-1zm-7-1a2 2 0 0 0-2 2v5a2 2 0 0 0 2 2h7a2 2 0 0 0 2-2V9a2 2 0 0 0-2-2h-7zm0-3a3.5 3.5 0 1 1 7 0v3h-1V4a2.5 2.5 0 0 0-5 0v3h-1V4z"/>
+                        </svg>                      
+                      </InputGroup.Text>
+                    </InputGroup.Prepend>
+                    <Form.Control
+                      name="password"
+                      type={this.state.passwordShown ? "text" : "password"}
+                      placeholder="Enter Password"
+                      aria-describedby="inputGroupPrepend"
+                      onChange={this.handleInputChange}
+                      required
+                    />
+                    <InputGroup.Append>
+                      <InputGroup.Text onClick={this.togglePasswordVisiblity}>
+                        <svg className="bi bi-eye-fill" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                          <path d="M10.5 8a2.5 2.5 0 1 1-5 0 2.5 2.5 0 0 1 5 0z"/>
+                          <path fill-rule="evenodd" d="M0 8s3-5.5 8-5.5S16 8 16 8s-3 5.5-8 5.5S0 8 0 8zm8 3.5a3.5 3.5 0 1 0 0-7 3.5 3.5 0 0 0 0 7z"/>
+                        </svg>
+                      </InputGroup.Text>
+                    </InputGroup.Append>
+                  </InputGroup>
+                </Form.Group>
+                <Form.Group>
+                  <Form.Check
+                    name="isRemember"
+                    label="Remember me"
+                    onChange={this.handleInputChange}
+                  />
+                </Form.Group>
+                <Button variant="dark" type="submit">SIGN IN</Button>
+              </Form>
+              <p class="mt-3"><a href="#" class="text-dark">Forgot Password?</a></p>
+            </Col>
+            <Col md={7} className="pl-5 my-auto homeRightSection">
+              <h1>Welcome Back :)</h1>
+              <h2>Let's do more with our data!</h2>
+              <Carousel className="mt-5">
+                <Carousel.Item className="homeCarousel">
+                  <img
+                    className="d-block w-100"
+                    src="carousel1.png"
+                    alt="First slide"
+                  />
+                </Carousel.Item>
+                <Carousel.Item className="homeCarousel">
+                  <img
+                    className="d-block w-100"
+                    src="carousel2.jpg"
```

Comment:
![image](https://user-images.githubusercontent.com/43271693/83181685-cf473b00-a157-11ea-8879-78318c01fe29.png)
(if we choose to store images file in the res folder, importing the file as component make it easier if we want to change the image later)

Reply:
Actually, what is the benefit of doing this? I will still have to import the file as 
```import Carousel2 from './Carousel2.png';```
So similar to the old way, I still need to change the route path

Reply:
I think it will be easier in case the images need to be changed (it's not a very important issue)
it's also fine without importing